server/src/main/java/org/jboss/as/server/controller/git/ElytronClientCredentialsProvider.java

```diff
@@ -39,12 +39,12 @@
  * Currently only login/password is supported.
  * @author Emmanuel Hugonnet (c) 2018 Red Hat, inc.
  */
-public class ElytronClientCredentialsProvider extends CredentialsProvider {
+class ElytronClientCredentialsProvider extends CredentialsProvider {
```

Comment:
@darranl, Is this class considered API? can the restriction of the visibility impact user code or just server code?

Reply:
Server code.